plugins/rocksdb/src/main/java/tech/pegasys/pantheon/plugin/services/storage/rocksdb/RocksDBKeyValuePrivacyStorageFactory.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.plugin.services.storage.rocksdb;
+
+import tech.pegasys.pantheon.metrics.MetricsSystem;
+import tech.pegasys.pantheon.plugin.services.storage.SegmentIdentifier;
+
+import java.nio.file.Path;
+import java.util.List;
+
+@Deprecated
```

Comment:
why is it deprecated?

Reply:
The Privacy solution is supposed to be using the same datasource as the main one ...there's a JI about doing that work, when complete this class should be deleted.

Reply:
👍 I wished `@Deprecated` annotation could take comments.